src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/AdoNet/SqlClient/SqlClientConstants.cs

```diff
@@ -0,0 +1,28 @@
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.SqlClient
+{
+    internal static class SqlClientConstants
+    {
+        public const string SqlCommandIntegrationName = "SqlCommand";
+
+        public static class SystemData
+        {
+            public const string AssemblyName = "System.Data";
+            public const string SqlClientAssemblyName = "System.Data.SqlClient";
+            public const string SqlCommandType = "System.Data.SqlClient.SqlCommand";
+            public const string MinimumVersion = "2.0.0";
```

Comment:
For CallTarget instrumentation, we can bump this back up to `"4.0.0"` and the tests (even the .NET Framework 2.0 test) should still work. That was just a workaround for the callsite instrumentation

Reply:
Remind me, was this due to type forwarding? The call in IL was to assembly 2.0.0 but at run time 4.0.0 was called instead?

Reply:
No, it didn't have to do with type forwarding per-se.

For callsite instrumentation, what happened was that we would look at an assembly (e.g. `App.dll`) and look at its assembly references to decide if any of its methods would need instrumentation If we were running on .NET 4.5+ (profiler requirement), then even assembly references to  `System.Data 2.0.0.0` should be instrumented because the .NET Framework will end up loading `System.Data 4.0.0.0`.

With CallTarget instrumentation, we're not instrumenting from the outside, but from the inside. So at the time that `System.Data.SqlClient.SqlCommand.ExecuteXXX` is being JIT compiled, we will make the decision to instrument. And this should always be `System.Data 4.0.0.0`. If it was `System.Data 2.0.0.0` that means assemblies from .NET Framework 2.x or 3.5 were loaded and we should not try to instrument.

Reply:
> The call in IL was to assembly 2.0.0 but at run time 4.0.0 was called instead?
Sorry I should have finished reading to the end. Yes, basically this. But with calltarget we already know that the executing assembly is 4.0.0.0 and never 2.0.0.0

Reply:
Sure, done in 70318d8